tripgo.swagger.yaml

```diff
@@ -138,6 +138,61 @@ paths:
       responses:
         200:
```

Comment:
@MarianoTucat I'm getting "Not a valid response definition" error here. What could that be?